modules/util/portal-tools-service-builder-test-service/service.xml

```diff
@@ -31,4 +31,56 @@
 			<finder-column comparator="&lt;" name="bigDecimalValue" />
 		</finder>
 	</entity>
+	<entity local-service="true" name="LocalizedEntry" remote-service="false">
+
+		<!-- PK fields -->
+
+		<column name="localizedEntryId" primary="true" type="long" />
+
+		<!-- Localized entity -->
+
+		<localized-entity>
+			<localized-column name="title" />
+			<localized-column name="content" />
+		</localized-entity>
+	</entity>
+	<entity local-service="true" name="LVEntry" remote-service="false" versioned="true">
+
+		<!-- PK fields -->
+
+		<column name="lvEntryId" primary="true" type="long" />
```

Comment:
"lvEntryId" is the one I'm really not sure about because we don't have that pattern in other places.  DLFileEntry has fileEntryId, MBMessage has messageId etc.